runtime/redirector/redirector.c

```diff
@@ -535,8 +536,13 @@ chooseJVM(JavaVMInitArgs *args, char *retBuffer, size_t bufferLength)
 	char *xmxstr = NULL;
 	U_64 requestedHeapSize = 0;
 
-	/* the command line is handled below but look into the ENVVAR_IBM_JAVA_OPTIONS here, since it is a special case */
-	envOptions = getenv(ENVVAR_IBM_JAVA_OPTIONS);
+	/* the command line is handled below but look into the ENVVAR_OPENJ9_JAVA_VERSION here, since it is a special case.
```

Comment:
Should we have `ENVVAR_OPENJ9_JAVA_OPTIONS` here?

Reply:
Good catch - took me a minute to see the difference between `_VERSION` & `_OPTIONS`